tests/yam/validate/validate_migration_logs.pm

```diff
@@ -15,6 +15,15 @@ sub run {
     select_console 'root-console';
 
     upload_logs("/var/log/distro_migration.log", failok => 1);
+    script_run 'tar zcvf /tmp/cache_wicked_config.tar.gz /var/cache/wicked_config/*';
+    upload_logs("/tmp/cache_wicked_config.tar.gz", failok => 1);
+    script_run 'tar zcvf /tmp/cache_udev_rules.tar.gz /var/cache/udev_rules/*';
+    upload_logs("/tmp/cache_udev_rules.tar.gz", failok => 1);
+    if (script_run("test -d /etc/udev/rules.d") == 0) {
```

Comment:
do we need to use a condition when we use `failok` ?

Reply:
Removed condition and update VR.